File: 1072.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = Number(lines.shift());
var dentro = 0;var fora = 0;
for(var i = 0; i < n; i++){
    var l = Number(lines.shift());
    if (l >= 10 && l <= 20){
        dentro = dentro + 1}
    else{
        fora += 1}}
console.log(`${dentro} in`)
console.log(`${fora} out`)

File: 1072.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    int n, dentro, fora, l;
    cin >> n;
    dentro = fora = 0;
    for(int i = 0; i < n; i++){
        cin >> l;
        if(l >= 10 and l <= 20){
            dentro += 1;
        }
        else{
            fora += 1;
        }
    }
    cout << dentro << " in\n" << fora << " out" << endl;
}
